lib/mayaUsd/fileio/utils/meshReadUtils.cpp

```diff
@@ -99,7 +99,7 @@ bool addCreaseSet(
     // .../lib/python2.7/site-packages/maya/app/general/creaseSetEditor.py
 
     MObject creasePartitionObj;
-    *statusOK = UsdMayaUtil::GetMObjectByName(":creasePartition", creasePartitionObj);
+    *statusOK = UsdMayaUtil::GetMObjectByName(MString(":creasePartition"), creasePartitionObj);
```

Comment:
Hopefully we don't need this change.

Reply:
I'll try to remove the overload taking a std::string and only keep the MString one, since that its the type that is needed.

Reply:
I tried removing the overload and the result is worse: many files hold std::string and would need to be modified.